projects/plugins/jetpack/tools/webpack.config.js

```diff
@@ -130,7 +130,7 @@ module.exports = [
 		],
 		output: {
 			...sharedWebpackConfig.output,
-			filename: '[name].min.js', // @todo: Fix this.
+			filename: '[name].js', // @todo: Fix this.
 		},
```

Comment:
The `@todo` comment is being resolved with this PR. 🙂 And since `filename: '[name].js'` is already [the default](https://github.com/Automattic/jetpack/blob/77b0fb5cd4d39046736fa0aa11726d3f3b8d44b8/projects/js-packages/webpack-config/src/webpack.js#L100), you should be safe to delete all of lines 131–134 and let line 122 above handle it.

Reply:
Thank you, updated!